### PR TITLE
bspwm : fix hardcoded /tmp path problem

### LINKED ISSUE #658: bspwm: Using hardcoded path `/tmp`
Just like what [here](https://www.reddit.com/r/termux/comments/gqi3j5/bspwm_problems_with_tmp_folder/frsyidq?utm_medium=android_app&utm_source=share&context=3) and [here](https://www.reddit.com/r/termux/comments/gqi3j5/bspwm_problems_with_tmp_folder/frukmfn?utm_medium=android_app&utm_source=share&context=3) said , we won't able to start bspwm because `/tmp` is not accessible.  I think we should write a patch to solve this issue

### PR DESCRIPTION
**Pull request by NoNameMan1231**
@xtkoba Did i done right? Because I'm first time writing a `.patch` file

Fixed #658

### CODE REVIEW

**Comment by NoNameMan1231** [comment]
hmm...

File: packages/bspwm/common.patch [hunk 1]
```diff
@@ -0,0 +1,11 @@
+--- src/common.h	2020-08-02 15:51:32.000000000 +0800
++++ src.mod/common.h	2022-03-21 21:24:04.760854462 +0800
```
Comment (Grimler91) [inline packages/bspwm/common.patch:2]: Use `./src/common.h`, and `./src.mod/common.h` to hopefully make it find the file

File: packages/bspwm/common.patch [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+@@ -25,7 +25,7 @@
+ #ifndef BSPWM_COMMON_H
+ #define BSPWM_COMMON_H
+ 
+-#define SOCKET_PATH_TPL  "/tmp/bspwm%s_%i_%i-socket"
++#define SOCKET_PATH_TPL  "/data/data/com.termux/files/usr/tmp/bspwm%s_%i_%i-socket"
```
Comment (Grimler91) [inline packages/bspwm/common.patch:8]: Please replace `/data/data/com.termux/files/usr` with `@TERMUX_PREFIX@`, it is automatically sed'ed when the [patch is applied](https://github.com/termux/termux-packages/blob/master/scripts/build/termux_step_patch_package.sh#L27)

**Review by Grimler91 — CHANGES_REQUESTED** [review]

File: packages/bspwm/common.patch (hunk 2 above)
Comment (NoNameMan1231) [reply packages/bspwm/common.patch:8]: Helpers.c also same?

File: packages/bspwm/common.patch (hunk 1 above)
Comment (NoNameMan1231) [reply packages/bspwm/common.patch:2]: Helpers.c also same?

**Comment by wmcbtech30** [comment]
What command did you create a patch? you seem to incorrectly created a patch or something else

File: packages/bspwm/common.patch (hunk 2 above)
Comment (Grimler91) [reply packages/bspwm/common.patch:8]: Yes

File: packages/bspwm/common.patch (hunk 1 above)
Comment (Grimler91) [reply packages/bspwm/common.patch:2]: Yes

**Comment by NoNameMan1231** [comment]
> What command did you create a patch? you seem to incorrectly created a patch or something else

`diff `

**Comment by NoNameMan1231** [comment]
Yay! Seems like it work!

**Comment by Grimler91** [comment]
Thanks!

**Pull request merged**